code/modules/mob/living/carbon/human/life.dm

```diff
@@ -133,10 +133,17 @@
 							   "my dad own this station",
 							   "the CHef put [pick("PROTEIN", "toiret waTer", "RiPPleing TendIes", "Einzymes","HORRY WALTER","nuTriments","ReActive MutAngen","TeSLium","sKrektonium")] in my [pick("wiSh soup","Bullito","rAingurber","sOilent GREEn","KoI Susishes","yaya")]!",
 							   "the monkey have TASER ARMS!",
-							   "qM blew my points on [pick("cOMbat Shtogun","inSuLated gloves","LOTS MASSHEEN!")]",
+							   "qM blew my points on [pick("cOMbat Shtogun","inSuLated gloves","LOTS MASSHEEN!", "peNTa Smmmmmmmm")]",
 							   "EI'NATH!",
 							   "WAKE UP SHEEPLES!",
-							   "et wus my [pick("wittle brother!!","fiancee","friend staying over","entiRe orphanage","love interest","wife","husband","liTTle kids","sentient cAT","accidentally")]!")
+							   "et wus my [pick("wittle brother!!","fiancee","friend staying over","entiRe orphanage","love interest","wife","husband","liTTle kids","sentient cAT","accidentally")]!",
+							   "magnazin is just an plate fur gun foond",
+							   "WHERE RECHAN BATONG?",
```

Comment:
```suggestion
							   "WHERE I CHARGE BATONG?",
```

![image](https://user-images.githubusercontent.com/12197162/113625472-e2b91180-9658-11eb-94fb-eabca5194e8a.png)
just so the reference is more closely followed